app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -394,14 +387,33 @@ docker run -d --name kong \
 [Declarative Configuration format]: /gateway/{{page.kong_version}}/reference/db-less-and-declarative-config/#the-declarative-configuration-format
 [Docker Volume]: https://docs.docker.com/storage/volumes/
 
+### Get started with Kong Gateway
+
+Now that you have a running Gateway instance, Kong provides a series of
+[getting started guides](/gateway/{{page.kong_version}}/get-started/comprehensive/)
+to help you set up and enhance your first Service.
+
+If you use the sample `kong.yml` in this guide, you already have a Service and
+a Route configured. Here are a few more things to check out:
```

Comment:
Should we mention that they need to choose the `Using decK (YAML)` tab once they follow a link?

Reply:
Scrap that, I have a PR open to automatically select the correct tab based on the URL at https://github.com/Kong/docs.konghq.com/pull/3616

Reply:
Have a commit ready to go for when the tab PR gets merged. I'll push + rebase after.